### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#60739]
#### VFS team name
Digitize Forms

#### VFS product name
Form 21-10210 Lay/Witness Statement

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
On /review-and-submit, the first accordion has an H3 of "Who is submitting this statement?", and the accordion contains two H4s that are also "Who is submitting this statement?"

This is confusing, especially for screen reader users who use the heading structure to help navigate the page.

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

### Platform Recommendation
I'm guessing this is a consequence of the way you presented those first conditional questions. Any way those H4s can be made unique relative to each other? Better still if both are different from the H3, but if we can at least rename the second one to something else that would go a long way.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by tbaker1026** [comment]
![Example.png](https://images.zenhubusercontent.com/63473b350a63bcf6bdef610f/efdd6c56-4361-44ec-89a4-ceaab0dc267e)![GoodExample.png](https://images.zenhubusercontent.com/63473b350a63bcf6bdef610f/55fb4d31-df5b-48e0-9cd9-0a7bc479ca8d)

**Comment by shiragoodman** [comment]
@tbaker1026 are you requesting validation of the fix from Governance team?

cc @briandeconinck

**Comment by tbaker1026** [comment]
@shiragoodman no just providing some samples for our developer to work on this issue in the next sprint - worked with Jeana on a fix

**Comment by tlei123** [comment]
We're dropping those confusing `<h4>`s instead of changing their texts; there'll just be no page-specific titles show in the expanded content anymore.

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/587583/3776665d-64e4-404b-8bec-9480edd39e69)

**Comment by tlei123** [comment]
PR merged.  Should be in Staging by ~7PM ET, Prod by tmrw (Tue) ~4 PM ET.

**Comment by tlei123** [comment]
@jeana-adhoc @tbaker1026 , bugfix is [now in Staging](https://staging.va.gov/supporting-forms-for-claims/lay-witness-statement-form-21-10210), ready for your validation.

**Comment by tlei123** [comment]
[2nd PR](https://github.com/department-of-veterans-affairs/vets-website/pull/25353) now opened for removing the rest of the unnecessary page-titles.

**Comment by tlei123** [comment]
@jeana-adhoc , bugfix is now in Staging.  Should be in Prod by ~3 E.

**Comment by tlei123** [comment]
@jeana-adhoc , 2nd PR now merged and [in Staging](https://staging.va.gov/supporting-forms-for-claims/lay-witness-statement-form-21-10210).
